src/dbally/similarity/elastic_store.py

```diff
@@ -0,0 +1,132 @@
+from typing import Dict, List, Optional, Tuple
+
+import numpy as np
+from elasticsearch import AsyncElasticsearch
+from elasticsearch.helpers import async_bulk
+
+from dbally.embedding_client.base import EmbeddingClient
+from dbally.similarity.store import SimilarityStore
+
+
+class ElasticStore(SimilarityStore):
+    """
+    The ElasticStore class stores text embeddings using knnSearch.
```

Comment:
Is it really knnSearch? What if the user specifies a different algorithm when initializing the class?